### PR TITLE
Please Ignore: Workflow Provided Discord Integrations

### PR DESCRIPTION
**Pull request by Rheeseyb**
Ignore for now as this will be used for testing at first

**Problem:**
The default webhook integration provided out of the box is far too noisy, and doesn't distinguish between issue comments and PR comments.

**Fix:**
Use `Ilshidur/action-discord@0.3.2` to customise the embeds and provide our own messages for each event.

### CODE REVIEW

**Comment by Rheeseyb** [comment]
An issue comment on a PR

File: .github/workflows/discord-issue-closed.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Discord Integration - Issue Closed
+on:
+  issues:
+    types: [closed]
```
Comment (Rheeseyb) [inline .github/workflows/discord-issue-closed.yml:4]: A review comment on a PR

Comment (Rheeseyb) [reply .github/workflows/discord-issue-closed.yml:4]: Another

**Review by seanparsons — CHANGES_REQUESTED** [review]
I need all of the changes.

**Review by seanparsons — APPROVED** [review]
All of the approvals.

**Review by seanparsons — CHANGES_REQUESTED** [review]
I'll need the TPS reports added to this.

**Review by seanparsons — APPROVED** [review]
Best TPS reports I've ever seen.

**Review by seanparsons — CHANGES_REQUESTED** [review]
Needs more workflow.

**Review by seanparsons — APPROVED** [review]
Discovered extra workflow behind the sofa.

**Pull request closed without merging**